README.md

```diff
@@ -97,7 +107,28 @@ Each lion should have:
 Create a new lion instance with the name `simba`
 
 ```ruby
-# code here
+class Animal
+	attr_accessor :name
+def initialize name
+	@name = name
+end
+def greet
+	puts "Hi my name is #{name}."
+end
+
+end
+
+
+class Lion < Animal
+	def name
+		if  #{name} == "Simba"
+		puts "true"
+	end
```

Comment:
close just missing the logic to set `king` attribute to a boolean depending on the value of a passed in`name`